### ISSUE TITLE
Fix Box releases

### ISSUE THREAD

**Issue by pxgamer** [pxgamer/arionum-cli#9]
This is waiting on a version of Box where the `paragonie/pharaoh` isn't using `dev-master`.

Waiting for: https://github.com/paragonie/pharaoh/pull/9

**Comment by pxgamer** [comment]
Looks like this has been merged into Box. Just waiting for a release.

PR: humbug/box#317
